src/screens/ImportSeedPhraseSheet.js

```diff
@@ -186,13 +168,42 @@ const ImportSeedPhraseSheet = ({ isEmpty, setAppearListener }) => {
     [setParams]
   );
 
-  const onPressImportButton = useCallback(() => {
+  const onPressImportButton = useCallback(async () => {
     if (isSecretValid && seedPhrase) {
-      return ConfirmImportAlert(({ color, name }) => {
-        if (color) setColor(color);
-        if (name) setName(name);
-        toggleImporting(true);
-      }, navigate);
+      const input = seedPhrase.trim();
+      let name = null;
+      // Validate ENS
+      if (isENSAddressFormat(input)) {
+        try {
+          const address = await web3Provider.resolveName(input);
+
+          if (!address) {
+            Alert.alert('This is not a valid ENS name');
```

Comment:
for the Alert.alert usage on this line + line 187, use `import { Alert } from 'react-native';`